dev/bots/prepare_package.dart

```diff
@@ -269,6 +154,7 @@ class ArchiveCreator {
     // unpacking it!) So, we use .zip for Mac, and the files are about
     // 220MB larger than they need to be. :-(
     final String suffix = platform.isLinux ? 'tar.xz' : 'zip';
+    stderr.writeln('os: $os version: $_version branch: $branchName suffix: $suffix');
```

Comment:
Was this a debug message, or intentional? If intentional why go to `stderr`?

Reply:
That was just a debug print.  Removed, thanks for seeing it.